config/initializers/open_uri_redirection.rb

```diff
@@ -2,7 +2,7 @@
 
 module OpenURI
   def self.redirectable?(uri1, uri2) # :nodoc:
-    uri1.scheme.downcase == uri2.scheme.downcase ||
+    uri1.scheme.casecmp(uri2.scheme) ||
```

Comment:
This is wrong. Output of casecmp is -1, 0, 1, not boolean. Need to `casecmp(...).zero?` for correct behaviour

Reply:
thanks for the catch... I actually meant to .zero it